### ISSUE TITLE
Request for a config option to increase Bellethorne damage

### ISSUE THREAD

**Issue by darchitectlp** [Vazkii/Botania#3315]
Any chance we can get a multiplier option for Bellethorne damage? I'd like it to be a solid late game defense in the pack I'm making, but it can't keep up with the mobs. Thanks for all the hard work; love this mod.

**Comment by ramirocantu** [comment]
I've implemented just that on my [fork](https://github.com/ramirocantu/Botania), I'll upload a release soon. Hopefully it'll get implemented, however I think that from a design perspective its kind of nasty to have just one block be configurable and also wrong to have every single block configurable from one file. Maybe in the future we can have config files for each block. In any case you can always modify the source files to your liking.

**Comment by ramirocantu** [comment]
As this is a request/suggestion I cannot submit a PR

**Comment by williewillus** [comment]
I'd like to ask you to please refrain from distributing modified jars without _clearly_ marking them as such (such as by changing the jar name, or changing the mod ID). 

Because someone on the internet is going to download it and get it confused, and bother us with supposed bugs. We'd really appreciate it.

As for a config option, not happening. Botania's philosophy is that excessive configurability impedes a cohesive experience. Ideas for rebalancing the flower as a whole can be discussed on Discord or on the subreddit.

**Comment by darchitectlp** [comment]
@williewillus Thanks for your reply. I'd still ask that you consider this at some point, as I don't think allowing configuration of flower rates, damage rates, etc. impedes the player's experience. Most players will never touch the config. However, for pack makers, this allows us to make Botania a late game alternative, or allows us to encourage the player to explore underused parts of the mod. Just my 2 cents. 

@ramirocantu THANK YOU! This is fantastic! Really appreciate the assist!

**Comment by dstecholution** [comment]
@darchitectlp my alt @denzuko does a bit of mod pack dev and found that short of writing mixin mods, kubejs or modtweaker is a good option for reconfiguring botana (and just about any other mod) to fill in where a pack maker would want that mod to be gated.

On a personal option. Botania while easy to get through the individual parts; is still very late game when one has to build infrastructure for botania or even remotely work towards the good stuff like red string and corporia. Been doing a 100 days challenge with GoG (Questbook edition) and on day 375 (casual with some afk) one still hasn't taken on the gaia guardian, so no end portal frames yet. Now is that on me, sure I do need to get gud and that's why this challenge exists but been playing botania since 1.7 days and even then it was a bit of mid-late game for the most average player.

tldr; kubejs or modtweaker solves the config issue for pack makers without breaking the game or mod for other players or devs.

**Comment by darchitectlp** [comment]
Awesome and thanks for pointing that out . Been living in 1.12 land for so long, wasn't even aware of what new things can be accomplished with kubejs etc. Will research!
